Source/Dafny/Rewriter.cs

```diff
@@ -2007,6 +2008,82 @@ class Induction_Visitor : BottomUpVisitor {
       }
     }
   }
-}
 
+  // ------------------- PrintEffectEnforcement -------------------
```

Comment:
Can this section be in a separate class/file?

Reply:
Move to a separate file in the `Resolver` folder.